### PR TITLE
Fix multiselect dragdrop behavior in ED trees.

### PR DESCRIPTION
**Pull request by alexatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

- Pressing CTRL while dragging now does not deselect rows and cancel the drag #768 
- Dropping ED from browser to browser does not require to drop on empty space now
- You can also multi drop EDs to create EUs now

### CODE REVIEW

**Review by lxatrhea — APPROVED** [review]
It fixes the drag problem, but not the extra user friendlyness of showing all the drop options. People still need to remember what a key combination exactly does. But the code looks good.

**Comment by alexatrhea** [comment]
Indeed, it fixes the current functionality and does not add the extra feature discussed there. As you mentioned some code from another branch would be needed but its a bit more tricky than that.

File: CDP4Composition/DragDrop/TreeListSelectionStrategyRowDragDrop.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        {
+        }
+
+        /// <summary>
+        /// Override of the Left mouse button down behavior
+        /// </summary>
+        /// <param name="hitInfo">The hitinfo</param>
+        protected override void OnAfterMouseLeftButtonDownCore(IDataViewHitInfo hitInfo)
+        {
+            if (Keyboard2.IsControlPressed)
```
Comment (samatrhea) [inline CDP4Composition/DragDrop/TreeListSelectionStrategyRowDragDrop.cs:54]: what is `keyboard2`? can we use a different name without the `2`?

**Review by samatrhea — CHANGES_REQUESTED** [review]
editorial change request - variable name change

File: CDP4Composition/DragDrop/TreeListSelectionStrategyRowDragDrop.cs (hunk 1 above)
Comment (alexatrhea) [reply CDP4Composition/DragDrop/TreeListSelectionStrategyRowDragDrop.cs:54]: Its a static class in the DevExpress that is advised to be used, can't change the name unfortunately.

**Review by samatrhea — APPROVED** [review]

**Comment by MartinFijneman** [comment]
verified with release testing 9.1.0 RC2
working as described for:

- Pressing CTRL while dragging now does not deselect rows and cancel the drag
- Dropping ED from browser to browser does not require to drop on empty space now
- You can also multi drop EDs to create EUs now

**Pull request merged**